include/data.h

```diff
@@ -112,7 +112,8 @@ struct TypeInfo
     u16 damageCategory:2;    // Used for B_PHYSICAL_SPECIAL_SPLIT <= GEN_3
     u16 useSecondTypeIconPalette:1;
     u16 isSpecialCaseType:1;
-    u16 padding:12;
+    u16 isHiddenPowerType:1; // Changing this for any type will change the distribution of all Hidden Power types from vanilla.
```

Comment:
Maybe it would be good to have the warning in `types_info,h` as well since this is the more likely place where users will look into it?

Reply:
Done 👍